src/components/Problems/Problems.tsx

```diff
@@ -169,13 +167,13 @@ export const Problems: FC = () => {
           <LoginForm closeDialog={closeLoginDialog} />
         </Stack>
       </Dialog>
-
-      <Stack direction="row" justifyContent="end" gap={5} mb={5}>
-        <Link variant="button1" href={rulesLink}>
-          Pravidlá
-        </Link>
-      </Stack>
       <Stack gap={5}>
+        <Typography variant="body1">
+          Ak si nevieš poradiť s niektorou z úloh, pozri si{' '}
+          <Link variant="button2" href={`/${seminar}/ako-riesit`}>
```

Comment:
`component="span"` by to mohlo spravit

Reply:
nah. ide o to, ze Link aj Button som spravil ako `display: "flex"` (definovane v `getButtonWrapperSx` v `buttonStyles.ts`, pouzite ako `...getButtonWrapperSx(...)` v `Link.tsx`). je pravda, ze `<Link>` (napr. z `next/link`) je bezne mysleny ako inline element (rovnako ako `<a>`), ale my to pouzivame hlavne ako samostatne stojace buttony... takze nejake moznosti ako to vyriesit:
1. ak to chceme niekde (napr. tu) ako inline, tak one-time prop na Link `sx={{display: "inline"}}` by mohol fungovat, ale neda sa zarucit, ze to nieco nerozbije (velkost elementu, ako sa sprava podciarknutie ako bottom border, a tak)
2. prepisat toto tak, aby to bol samostatne stojaci (nie inline) element, ale newrapovalo by sa to, takze by si trebalo byt isty, ze ten text bude na jeden riadok:
```ts
<Stack direction="row">
  <Typography variant="body1">
    Ak si nevieš poradiť s niektorou z úloh, pozri si{' '}
  </Typography>
  <Link variant="button2" href={`/${seminar}/ako-riesit`}>
</Stack>
```
3. vlastne este existuje `display: "inline-flex"`, co je asi vhodnejsie ako ten one-time override 😄 
4. mozno by sa ten `inline-flex` zvladol aj priamo v tych `buttonStyles.ts` styloch miesto flexu, ale co ja viem 😄 

volii by som 3. ak to funguje ako sa ocakava

Reply:
pouzila som to 3., u mna to funguje tak, ako sme si predstavovali